frozen-abi/Cargo.toml

```diff
@@ -27,6 +27,7 @@ bv = { workspace = true, features = ["serde"] }
 bytes = { workspace = true }
 dashmap = { workspace = true }
 log = { workspace = true, features = ["std"] }
+rand = { workspace = true }
```

Comment:
If we move this down into the `[target.'cfg(not(target_os = "solana"))'.dependencies]` block, and gate the new module on `[cfg(not(target_os = "solana"))]`, we can avoid the breaking change on Solana builds